packages/lesswrong/components/tagging/SubforumCommentsThread.tsx

```diff
@@ -16,6 +16,15 @@ const SubforumCommentsThread = ({ tag, terms, newForm=true }: {
     enableTotal: true,
   });
   
+  const sortByRef = useRef(terms.sortBy);
+  const orderedResults = useOrderPreservingArray(
+    results || [],
+    (comment) => comment._id,
+    // If the selected sort order changes, clear the existing ordering
+    sortByRef.current === terms.sortBy ? "interleave-new" : "no-reorder"
```

Comment:
This is to make it clear the ordering when you change the sort order, I thought about adding a dependency array to this hook instead of this but decided that was overengineering at this point